### ISSUE TITLE
select2_from_ajax_multiple: Method getKey does not exist 

### ISSUE THREAD

**Issue by mochilor** [Laravel-Backpack/CRUD#830]
I am experiencing problems with "select2_from_ajax_multiple". When current model has some related entities, it throws a `Method getKey does not exist` error.

After inspecting the code I discovered that Eloquent find() method is receiving a Model instance instead of an id (select2_from_ajax_multiple.blade.php, [line 19](https://github.com/Laravel-Backpack/CRUD/blob/master/src/resources/views/fields/select2_from_ajax_multiple.blade.php#L21)):

```blade
@foreach ($old_value as $result_key)
  @php
     $item = $connected_entity->find($result_key); // Here is the problem!
  @endphp
  // ...
```

I have solved it passing the model id attribute to the method:

```php
$item = $connected_entity->find($result_key->id);`
```

Hope it helps! I can submit a PR if you want.

PS: Backpack is a wonderful package!

**Comment by mochilor** [comment]
Well, now I've found that original code is needed in case of validation fail occurs. The problem is that `$old_value` can contain a Collection of models (when you first access edit page) or an array of integers (when you click save button and validation fails). So, a better solution may be to change [line 5](https://github.com/Laravel-Backpack/CRUD/blob/master/src/resources/views/fields/select2_from_ajax_multiple.blade.php#L5) with this (please, correct me if I'm wrong!):
```php
$old_value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value']->pluck('id')->toArray() : (isset($field['default']) ? $field['default'] : false ));
```
This way, `$old_value` always contains an array of id's.

**Comment by lloy0076** [comment]
I meddled a little in the code snippet to make it more readable; if that works, that's good but tha many ternaries does somewhat hurt my eyes 👿

**Comment by tabacitu** [comment]
Fixed in 3.2.20, please let me know if you're still having the issue after a ```composer update```.

@mochilor thanks for the bug report and the solution.
